symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +3029,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
```

Comment:
How are you going to use `hbar` in sympy? `sympy.physics` would import from sympy.core.backend which would in turn import from `sympy.physics` for `hbar` if `USE_SYMENGINE=1`.